### ISSUE TITLE
Undefined subroutine &pf::action::security_event_force_close when trying to close a security event from another one

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6048]
**Describe the bug**
If you try to use "close" action on a security event to close another one, you see following message in log:

```
Dec 16 14:01:45 pfcen7dev packetfence_httpd.webservices: httpd.webservices(1847) INFO: [mac:52:54:00:da:53:75] security event 3000010 added for 52:54:00:da:53:75 as open (pf::security_event::security_event_add)
Dec 16 14:01:45 pfcen7dev packetfence_httpd.webservices: httpd.webservices(1847) INFO: [mac:52:54:00:da:53:75] executing action 'close' on class 3000010 (pf::action::action_execute)
Dec 16 14:01:45 pfcen7dev packetfence_httpd.webservices: httpd.webservices(1847) INFO: [mac:52:54:00:da:53:75] SECURITY_EVENT_ID to close: 3000009 (pf::action::action_close)
Dec 16 14:01:45 pfcen7dev packetfence_httpd.webservices: httpd.webservices(1847) ERROR: [mac:52:54:00:da:53:75] Undefined subroutine &pf::action::security_event_force_close called at /usr/local/pf/lib/pf/action.pm line 387.
 (pf::WebAPI::JSONRPC::__ANON__)
```

**To Reproduce**
1. Create a security event that try to close "Generic" security event
2. Apply Generic security event on your node
3. Apply your new security event

=> You got two security events opened on your nodes:
- Generic security event (action failed)
- New security event used to close Generic security event
```ini
[3000011]
trigger=
actions=close
desc=close_generic
enabled=Y
vclose=1300000
access_duration=5m
```
**Expected behavior**
- Generic security event **closed**
- New security event used to close Generic security event **closed**

**Additional context**
I took a look at `action.pm` and import of `security_event_force_close` looks correct.

If I run:
```
perl -I/usr/local/pf/lib -Mpf::action -e 'pf::action::security_event_force_close("52:54:00:95:c9:25","1300000")'
perl -I/usr/local/pf/lib -Mpf::action -e 'pf::action::action_close("52:54:00:95:c9:25","1300000")'
```

Generic security event is correctly closed. So it means from my point of view that missing function is due to context. Perhaps because we call action.pm from `pf::WebAPI::JSONRPC::__ANON__`. 

In `action.pm`, I see a lot of `require package` at several places in the code.

**Comment by nqb** [comment]
A quick fix is to replace `security_event_force_close` by `pf::security_event::security_event_force_close` here: https://github.com/inverse-inc/packetfence/blob/9a469c35226b2ef9a9914321742ebba0e08b660a/lib/pf/action.pm#L387

but I'm sure there is an issue somewhere.

**Comment by jrouzierinverse** [comment]
Fixed by 5eaf5037dfa3c3f9b975ddb8be29bc13fc34f981

**Comment by nqb** [comment]
Has been backported to maintenance.
